docs/guides/composer.mdx

```diff
@@ -0,0 +1,731 @@
+
+### 4. Inspect your circuit, step-by-step
+
+The Inspect mode de-mystifies the inner workings of the circuits you create. It steps through a simulation of your circuit, one layer at a time, so that you can see the state of the qubits as the computation evolves.
+
+* In the **View** menu, select the panels for the visualizations that you want to use.
+
+* Click the **Inspect** toggle in the toolbar. Note that once the Circuit Inspector is toggled on, you cannot add any more operations until it is turned off again.
+
+* If you built your circuit with the Freeform alignment turned on, note that the Circuit Inspector automatically turns on the Left alignment.
+
+* To move step-by-step through visualizations of your circuit’s components, control the movement of the Circuit Inspector using the forward and rewind buttons.
+
+![Controls to step forward and backward, and to jump to a circuit's beginning or end.](/docs/images/guides/composer/5f2dfb7c-a244-405c-8950-f422a4408fcf.avif)
+
+Inspector controls, in order: jump to the beginning of a circuit, step backward, play from one break point to the next, step forward, jump to the end of a circuit.
+
+* To inspect only some operations, click the operations you want to be inspected, and a colored overlay appears over each that indicates they will be included when you run the Circuit Inspector. To unselect an operation, click on it again, and the overlay disappears.
+
+* To learn more about interpreting visualizations, see the [Visualizations](#visualizations) topic.
+
+* To close the Inspector and return to editing your circuit, click the Inspect toggle in the toolbar.
+
+<Admonition type="note" title="Randomness in the simulator">
+The simulator creates randomness by generating results based on a *seed*. The seed is the initial value introduced into the algorithm that generates pseudorandom numbers. You can see the seed number by selecting Visualizations seed from the Edit menu. You can also set the seed yourself by changing the value in the box.
+</Admonition>
+
+## Run circuits and view results
+
+Follow the steps below to run quantum circuits on QPUs or simulators, and view the results.
+
+### Choose your job settings
+
+Click **Run circuit** in the upper right corner. In the window that opens, select an available QPU. You can also choose an instance, which is associated with a plan (such as the Open, Flex, or Premium Plans). The instance you choose affects which QPUs are available to you. Click the View details link in the table of QPUs to see more information on each QPU.
+
+Next, you can set the number of shots (executions) of your circuit that the backend will perform.
+
+You can optionally name the job and add tags in this panel as well. This will not change the name of the circuit. A "Composer" tag has been pre-populated to make it easier to filter your Workloads table by Composer jobs. You can remove this tag.
```

Comment:
Is the tense right?

```suggestion
You can optionally name the job and add tags in this panel as well. This will not change the name of the circuit. A "Composer" tag will be pre-populated to make it easier to filter your Workloads table by Composer jobs. You can remove this tag.
```

Reply:
Rewrote